dynamic_instrumentation/src/point/build.rs

```diff
@@ -140,6 +151,15 @@ impl<'tcx> InstrumentationBuilder<'_, 'tcx> {
         self
     }
 
+    /// Sets the priority of this instrumentation. A lower value
+    /// signifies higher priority and implies that an instrumentation
+    /// with higher priority will be placed before one with lower
+    /// priority.
```

Comment:
Maybe put this documentation on `InstrumentationPriority`?  I'm not sure, but I feel like something should be there.